src/SignalR/clients/ts/signalr/src/Utils.ts

```diff
@@ -7,6 +7,10 @@ import { NullLogger } from "./Loggers";
 import { IStreamSubscriber, ISubscription } from "./Stream";
 import { Subject } from "./Subject";
 
+// Version token that will be replaced by the prepack command
+/** The version of the SignalR client. */
+export const VERSION: string = "0.0.0-DEV_BUILD";
```

Comment:
Why was this moved from Index to Utils?

Reply:
So that it could be referenced from other files. You can't really reference index.ts as it's exporting all the public types.